ops/model.py

```diff
@@ -3507,6 +3521,17 @@ def reboot(self, now: bool = False):
         else:
             self._run("juju-reboot")
 
+    def credential_get(self) -> 'CloudSpec':
+        """Access cloud credentials by running the credential-get hook tool.
+
+        Returns the cloud specification used by the unit's model.
```

Comment:
I think we can just say:

```suggestion
        Returns the cloud specification used by the model.
```